src/Neo/SmartContract/Native/NativeContract.cs

```diff
@@ -266,19 +266,21 @@ public ContractState GetContractState(IsHardforkEnabledDelegate hfChecker, uint
         /// </summary>
         /// <param name="settings">The <see cref="ProtocolSettings"/> where the HardForks are configured.</param>
         /// <param name="index">Block index</param>
-        /// <param name="hardfork">Active hardfork</param>
+        /// <param name="hardforks">Active hardforks</param>
         /// <returns>True if the native contract must be initialized</returns>
-        internal bool IsInitializeBlock(ProtocolSettings settings, uint index, out Hardfork? hardfork)
+        internal bool IsInitializeBlock(ProtocolSettings settings, uint index, out Hardfork[]? hardforks)
         {
             // If is not configured, the Genesis is the a initialized block
             if (index == 0 && ActiveIn is null)
             {
-                hardfork = null;
+                hardforks = null;
                 return true;
```

Comment:
I dont see this being a problem.

Reply:
It is a problem since we still need to perform initialization in the described case, otherwise initial storage items will be missing in the contracts. For example, `Initialize` won't be called at all for native GasToken:
https://github.com/neo-project/neo/blob/629baf819f9ebedec4b2e89e7b55a5adf1836597/src/Neo/SmartContract/Native/GasToken.cs#L29-L37

Reply:
@AnnaShaleva sorry interpreted

Reply:
Good point, fixed